src/class/eventManagement/incomer.class.ts

```diff
@@ -405,14 +400,23 @@ export class Incomer <
 
     if (this.incomerChannel) {
       await this.incomerChannel.publish({
-        name: "close",
+        name: "CLOSE",
         redisMetadata: {
           origin: this.providedUUID,
           incomerName: this.name,
           prefix: this.prefix
         }
       });
     }
+    else {
+      const oldTransactions = await this.defaultIncomerTransactionStore.getTransactions();
+
+      await Promise.all(
+        [...oldTransactions.entries()]
+          .filter(([__, transaction]) => transaction.name === "REGISTER")
+          .map(([id]) => this.defaultIncomerTransactionStore.deleteTransaction(id))
```

Comment:
Maybe put everything in the `map` with a `if` on the `trnasaction.name`. It avoid to loop twice, in case where the array is long ?